<!-- {about} --><a name="about"></a>

  <p id="about"><em>Oscheius tipulae</em> is a hermaphroditic species and member of the Rhabditidae [1]. The species has been used as a model system to study vuvla formation [1].  The sequenced strain, CEW1, was originally isolated in Brazil [2]. See below for more information.</p>


<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
  <h2 id="assembly" style="margin-bottom:5px">Species and strain</h2>
  <p><em>O. tipulae</em> was formally described in [3].

  <p>CGC strain entry: <a href="https://cgc.umn.edu/strain/CEW1">CEW1</a></p>

<a name="assembly"></a>
<h2 id="assembly" style="margin-bottom:5px">Raw data</h2>
<p>The raw data can be downloaded from either ENA or NCBI:</p>
<p><a href="https://www.ebi.ac.uk/ena/data/view/PRJEB15512">https://www.ebi.ac.uk/ena/data/view/PRJEB15512</a></p>
<p><a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJEB15512">https://www.ncbi.nlm.nih.gov/bioproject/PRJEB15512</a></p>


<a name="assembly"></a>
<h2 id="assembly" style="margin-bottom:5px">Assembly</h2>
<p>The genome of <em>O. tipulae</em> was assembled by Georgios Koutsovolous and Fabrice Besnard (Blaxter Lab) using a combination of short-read and mate-pair data. It was published in Augustus 2017 [2].</p>
<p>From [2]:</p>
<p>"Raw reads were trimmed for adaptors using Cutadapt (Martin 2011) and low-quality bases, then corrected for sequencing errors based on k-mer content using Quake (Kelley et al. 2010) and JELLYFISH (Marçais and Kingsford 2011). Raw data were checked with FastQC (Andrews 2010) and a preliminary assembly generated with CLC Assembly Cell (CLC bio 2017) (Table S5). The CLC assembly was screened for contaminants using taxon-annotated, GC-coverage (TAGC) plots (Kumar et al. 2013). Only data deriving from E. coli, the food source, was identified as contaminant and the corresponding reads were removed. The optimal k-mer size for assembly of the cleaned read set was estimated using KmerGenie (Chikhi and Medvedev 2014). Nine different assemblers (Table 1) were used to generate preliminary assemblies and these were assessed using basic metrics, correctness of read alignment using ALE (Clark et al. 2013) and REAPR (Hunt et al. 2013), and biological completeness using Core Eukaryotic Genes Mapping Appoach (CEGMA) (Parra et al. 2007) and direct identification of ribosomal RNA genes and mitochondrial genome sequences. SPAdes (Bankevich et al. 2012) outperformed the other assembly tools in almost all aspects and was chosen as draft assembly nOt.1.0. An improved assembly limited to the nuclear genome (nOt.2.0) was generated by removing mitochondrial contigs and contigs of abnormally low coverage and by breaking all scaffolds where mis-assembly had been indicated from analysis of mapping plots and REAPR fragment coverage distribution (FCD) scores."</p>
<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
  <h2 id="annotation" style="margin-bottom:5px">Annotation</h2>
  <p>From [2]:</p>
  <p>"Genes were predicted using a two-pass pipeline (Koutsovoulos et al. 2014) (see Figure S3) based on MAKER2 (Holt and Yandell 2011) and Augustus (Stanke et al. 2006), and using the transcriptome data as evidence. Repeats were identified in the assembly using RepeatModeler (Smit et al. 2013–2015). MAKER2 was run in an SGE cluster using the SNAP (Korf 2004) ab initio gene finder trained by CEGMA (Parra et al. 2007) output models, the GeneMark-ES ab initio finder, SwissProt proteins, and O. tipulae transcripts. The transcriptome data were filtered so that only reads >300 bases that had significant basic-local-alignment-search-tool (BLAST) similarity to C. elegans protein databases were kept. The MAKER2 predictions were used to train Augustus to generate a custom gene-finding profile for O. tipulae. Finally, Augustus was used with the gene-finder profile and O. tipulae transcripts to predict the final gene set. Not enough transcript evidence was available to train a model of untranslated regions (UTRs), and therefore no UTRs were annotated."</p>
<!-- {annotation} -->
<!-- {references} -->
  <h2>References</h2>
  <ol>
    <li><a id="ref-01"></a><a href="http://www.wormbook.org/chapters/www_genomesOtipulae/genomesOtipulae.html">WormBook: <em>Oscheius tipulae</em></a>.</li>
    <li><a id="ref-02"></a><a href="http://www.genetics.org/content/206/4/1747">Toward Universal Forward Genetics: Using a Draft Genome Sequence of the Nematode <em>Oscheius tipulae</em> To Identify Mutations Affecting Vulva Development</a>.</li>
    <li><a id="ref-03"></a><a href="http://booksandjournals.brillonline.com/content/journals/10.1163/187529293x00187">Redescription of Rhabditis (Oscheius) Tipulae (Nematoda: Rhabditidae) Associated With Leatherjackets, Larvae of Tipula Paludosa (Diptera: Tipulidae)</a>.</li>
  </ol>
 <!-- {references} -->
